### PR TITLE
Basic implementation of the section to show the team

### PR DESCRIPTION
**Pull request by bahorn**
Need to get Pictures of all of us now and make it dynamically load the images, but enough for a preview.

### CODE REVIEW

**Comment by bahorn** [comment]
Currently just need to add social media links then the features are done.

Will need some style improvements and fixes to our bios.

File: src/components/Team/Team.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  )
+  return (
+    <Row className="row-eq-height">
+      {props.team
+        .sort((a, b) => a.listOrder - b.listOrder)
+        .map(member => (
+          <Col md="12" lg="4" key={member.id} className="mb-3">
+            <Card className="shadow-sm bg-white h-100">
+              <CardBody>
+                <Img fluid={allFiles[member.name]} style={{ margin: '0.5em' }} />
```
Comment (jmsv) [inline src/components/Team/Team.js]: remove margin or just use `{ margin: '0.5em 0'}` for top and bottom margin

**Review by jmsv — CHANGES_REQUESTED** [review]

File: src/components/Team/Team.js (hunk 1 above)
Comment (bahorn) [reply src/components/Team/Team.js]: Done!

**Review by jmsv — APPROVED** [review]

**Comment by bahorn** [comment]
Now just need to wait for the rest to correct their bios and social medias.

**Pull request merged**